File: w10/d02/Corey_Leveen/morning_substring.js
function find_substring (string, substring) {
  var count = 0;

  for (var i = 0; i < string.length+1; i++) {
    if (string[i] == substring[count]) {
      count++;
    }
    else {
      count = 0;
    }

    if (count == substring.length) {
      return (i - count) + 1;
    }

    else if (i == string.length) {
      return false;
    }
  }
}

File: w10/d02/Corey_Leveen/morning_substring.rb
def find_substring(string, substring)

  count = 0
  i = 0
  while i < string.length + 1
    if string[i] === substring[count]
      count += 1
    else
      count = 0
    end

    if count === substring.length
      return ((i - count)+1)
    elsif i === string.length
      return false
    end
    i += 1
  end
end
